deepchecks/utils/features.py

```diff
@@ -219,13 +220,22 @@ def _calc_importance(
         raise errors.DeepchecksTimeoutError('Permutation importance calculation was not projected to finish in'
                                             f' {permutation_importance_timeout} seconds.')
 
+    if hasattr(model, 'score'):
+        if hasattr(model, 'fit'):
+            scoring = None
+        else:
+            scoring = model.score
+    else:
+        scoring = scorer.scorer
+
     r = permutation_importance(
         model,
         dataset_sample.features_columns,
         dataset_sample.label_col,
         n_repeats=n_repeats,
         random_state=random_state,
-        n_jobs=-1
+        n_jobs=-1,
+        scoring=scoring
```

Comment:
if this is scorer from the model (model.score) it doesn't make sense we did the calculation of time on our default scorer